ext/bg/js/dictionary.js

```diff
@@ -159,11 +159,14 @@ function dictTermsGroup(definitions, dictionaries) {
 }
 
 function dictTermsMergeBySequence(definitions, mainDictionary) {
-    const definitionsBySequence = {'-1': []};
+    const sequencedDefinitions = new Map();
+    const nonSequencedDefinitions = [];
     for (const definition of definitions) {
-        if (mainDictionary === definition.dictionary && definition.sequence >= 0) {
-            if (!definitionsBySequence[definition.sequence]) {
-                definitionsBySequence[definition.sequence] = {
+        const sequence = definition.sequence;
+        if (mainDictionary === definition.dictionary && sequence >= 0) {
+            sequencedDefinition = sequencedDefinitions.get(sequence);
```

Comment:
`let`

Reply:
I forgot that eslint won't catch that due to `"no-undef": "off"`